include/CppUTest/StandardCLibrary.h

```diff
@@ -1,8 +1,10 @@
 
+/* Must include this first to ensure the StandardC include in CppUTestConfig still happens at the right moment */
+#include "CppUTestConfig.h"
```

Comment:
It doesn't make much sense when you actually want StandardC :)

Anyways, do you know any situation in which the current solution leads to problems?